WebClient/pom.xml

```diff
@@ -11,78 +11,18 @@
     <modelVersion>4.0.0</modelVersion>
     <artifactId>web-client</artifactId>
     <packaging>war</packaging>
-
-    <properties>
-        <webapp-runner.version>8.0.33.1</webapp-runner.version>
-        <servlet-api.version>2.5</servlet-api.version>
-        <spring-core.version>4.3.2.RELEASE</spring-core.version>
-        <spring-web.version>4.3.2.RELEASE</spring-web.version>
-        <runnable.version>2.3</runnable.version>
-    </properties>
-
     <dependencies>
```

Comment:
USUALLY (just usually, nothing serious, feel free to ignore, disregard, trample, etc.) dependencies come AFTER properties. But, you know, that's just, common sense. Clearly, you're not common. ;P